app/lib/esClient.js

```diff
@@ -0,0 +1,80 @@
+const log = require('./logger');
+const elasticsearch = require('elasticsearch');
+const esConfig = require('../../config/config').es;
+
+const client = elasticsearch.Client({ host: `${esConfig.host}:${esConfig.port}` });
+
+function getBaseQuery(size) {
+  return {
+    index: esConfig.index,
+    type: 'pharmacy',
+    body: {
+      size,
+      query: {
+        bool: {}
+      }
+    }
+  };
+}
+
+function build(location, radius, size) {
+  const query = getBaseQuery(size);
+
+  query.body.query.bool.filter = {
+    geo_distance: {
+      distance: `${radius}mi`,
```

Comment:
It is worth extracting `mi` to a variable so it can be used here and on https://github.com/nhsuk/nearby-services-api/pull/91/files#diff-a132c3bac9062e1773bdf0cbf8e723ddR41 to avoid it be missed when one is changed?